include/icon.h

```diff
@@ -11,6 +11,16 @@ struct mako_icon {
 	cairo_surface_t *image;
 };
 
+struct iiibiiay {
```

Comment:
Eh. `aiii(siosssss(si)iiiiii)iiiba{yys}a`?

(Translation from D-Bus to English: can we change this to a human-readable name?)

Reply:
Not sure what I should name it. Any suggestion?

Reply:
`mako_icon_data`?

Reply:
Done!